mm2src/db_common/src/sqlite.rs

```diff
@@ -346,9 +417,25 @@ fn validate_ident_impl<F>(ident: &str, is_valid: F) -> SqlResult<()>
 where
     F: Fn(char) -> bool,
 {
+    let validation_error = || {
+        SqlError::SqliteFailure(
+            rusqlite::ffi::Error {
+                code: rusqlite::ErrorCode::ApiMisuse,
+                extended_code: rusqlite::ffi::SQLITE_MISUSE,
+            },
+            None,
+        )
+    };
+
+    if ident.as_bytes()[0].is_ascii_digit() {
+        log::error!("{ident} starts with number.");
+        return Err(validation_error());
+    }
```

Comment:
I think we should add a check that ident is not an empty str

Reply:
Done https://github.com/KomodoPlatform/komodo-defi-framework/compare/c83dffff426d92b72d0ec77f8598c003439df2d7..6cbfb21089edffd7adf51ed563f5e2fd24f80afd